test/docker_e2e.sh

```diff
@@ -78,6 +78,16 @@ docker run --rm --name rid-db-manager \
 	--db_version v3.0.0 \
 	--cockroach_host crdb
 
+sleep 1
+echo "Bootstrapping SCD Database tables"
+docker run --rm --name scd-db-manager \
+	--link dss-crdb-for-debugging:crdb \
+	-v $(pwd)/build/deploy/db-schemas/scd:/db-schemas/scd \
+	local-db-manager \
+	--schemas_dir db-schemas/scd \
+	--db_version v1.0.0 \
```

Comment:
Keeping track of these things are annoying for our test infra. I'll add an issue to add flag to always upgrade to latest in test

Reply:
Added #354